config/default.yaml

```diff
@@ -49,8 +47,8 @@ documentUpload:
     totalSupportingDocuments: 5
     totalAdditionalDocuments: 20
     totalOtherInformation: 20
-    sizeInBytes: 20480000
-    multimediaSizeInBytes: 104857600
+    sizeInBytes: 524288000
+    multimediaSizeInBytes: 524288000
```

Comment:
Could we just have one value since all types of files have the same max upload size?